ERCS/erc-7826.md

```diff
@@ -0,0 +1,553 @@
+---
+eip: 7826
+title: Quantum Supremacy Bounty
+description: A smart contract to prove a strict quantum supremacy and indicate requiring quantum-secure verification schemes.
+author: Nicholas Papadopoulos (@nikojpapa)
+discussions-to: https://ethereum-magicians.org/t/erc-7826-quantum-supremacy-bounty/21866
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-06-26
+requires: 2470
+---
+
+## Abstract
+
+This proposal introduces a smart contract containing a classically intractable puzzle that is expected to only be able to be solved using quantum computers.
+The contract is funded with ETH, which can only be retrieved by solving the problem.
+On-chain applications can then watch this contract to be aware of the quantum advantage milestone of solving this puzzle.
+For example, Ethereum smart contract wallets can, using custom verification schemes such as those based on [ERC-4337](./erc-4337.md), watch this contract and fall back to a quantum secure signature verification scheme if and when it is solved.
+
+The contract, then, serves the two purposes of (1) showing proof of a strict quantum supremacy[^1] that is strong enough to 
+indicate concerns in RSA and ECDSA security, and (2) acting as an indicator to protect Ethereum assets by triggering quantum-secure 
+signature verification schemes.
+
+## Motivation
+
+Quantum supremacy[^1] is a demonstration of a quantum computer solving a problem that would take a classical computer an infeasible amount of time to solve.
+Previous attempts have been made to demonstrate quantum supremacy, e.g. Kim[^2], Arute[^3] and Morvan[^4], 
+but they have been refuted or at least claimed to have no practical benefit, e.g. Begusic and Chan[^5], Pednault[^6], 
+and a quote from Sebastian Weidt (The Telegraph, "Supercomputer makes calculations in blink of an eye that take rivals 47 years", 2023).
```

Comment:
Please cite external documents using the formats allowed in EIP-1. If you cannot, please either propose a new origin using the process from EIP-5757 or remove this link.